doc/code/qml_transforms_qcut.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
+
+qml.transforms.qcut
+===================
```

Comment:
Hey Tom, you could put this content in the `qcut/__init__.py` module docstring, and avoid needing to have a separate file in `doc/code`.

The advantage here is that is closer to the code 🙂 That way anyone who modifies the `qcut` module can more easily see that they have to update this.

Similar to how the devices subpackage works: https://github.com/PennyLaneAI/pennylane/blob/1bca151b7f3172237654e18fcf42749c9aa5d149/pennylane/devices/__init__.py#L15-L32